aten/src/ATen/gen.py

```diff
@@ -189,6 +189,7 @@ def check_all_files_written(self):
     ('Long', 'int64_t', 'Long', False),
     ('Short', 'int16_t', 'Long', False),
     ('Half', 'Half', 'Double', True),
+    ('QInt8', 'qint8', 'Long', False),
```

Comment:
do we actually need 'qint8' and 'Long' defined here?

Reply:
what do you mean? we do need types like CPUQInt8Type.